cfme/cloud/keypairs.py

```diff
@@ -184,6 +185,41 @@ def download_private_key(self):
         view.toolbar.configuration.item_select('Download private key')
         view.flash.assert_no_error()
 
+    def set_ownership(self, owner=None, group=None, cancel=False, reset=False):
+        """Set instance ownership
+
+        Args:
+            user (User): user object for ownership
```

Comment:
This is inaccurate - you're passing the group description, a string.

I think it is better to actually take user/group objects, and reference the required field here - instead of forcing the caller to know which attribute of the object needs to be passed.

Reply:
Done